server/src/infra/repositories/api-key.repository.ts

```diff
@@ -3,9 +3,10 @@ import { Injectable } from '@nestjs/common';
 import { InjectRepository } from '@nestjs/typeorm';
 import { Repository } from 'typeorm';
 import { APIKeyEntity } from '../entities';
+import { DummyValue, GenerateSql } from '../infra.util';
 
 @Injectable()
-export class APIKeyRepository implements IKeyRepository {
+export class ApiKeyRepository implements IKeyRepository {
```

Comment:
If we are going to rename this to API should we also do the same for the entity and other other places we have this? Personally I don't mind whether it is API or Api

Reply:
True, I changed it because I take the name and convert it to `api.repository.sql`. The algorithm didn't work if it was `API`. I can look at changing the other references in the future to match this one.